ui/src/integrations/ethereum/contracts/FleekERC721.json

```diff
@@ -1,9 +1,88 @@
 {
-  "timestamp": "2/15/2023, 12:43:39 PM",

```

Comment:
I have a question about this file. I think we have an identical version of this in `contracts` so is it necessary to have it here as well?

Reply:
Yeah, we need it for the integration